### ISSUE TITLE
Error in IE11 - Multiple definitions of a property not allowed in strict mode

### ISSUE THREAD

**Issue by mrdigitalau** [wuori/vue-credit-card-validation#14]
Thanks for this sick plugin!

Issue using this in IE11.

Error received: Multiple definitions of a property not allowed in strict mode.

You have two functions with the same name in vue-credit-card-validation.js

```

    cardExpiryVal: function () {
        return vuePayment.payment.cardExpiryVal(this.value);
    },

    cardExpiryVal: function (value) {
        var ref = Array.from(value.split(/[\s\/]+/, 2));
        var month = ref[0];
        var year = ref[1];

        // Allow for year shortcut
        if (((year != null ? year.length : undefined) === 2) && /^\d+$/.test(year)) {
            var prefix = (new Date).getFullYear();
            prefix = prefix.toString().slice(0, 2);
            year = prefix + year;
        }

        month = parseInt(month, 10);
        year = parseInt(year, 10);

        return { month: month, year: year };
    },
```

**Comment by wuori** [comment]
@mrdigitalau Good catch! I'll try to have this updated soon.

**Comment by wuori** [comment]
@mrdigitalau Just published package v1.0.9
